### ISSUE TITLE
Let users request a list of their automatic subscriptions

### ISSUE THREAD

**Issue by rossjrw** [croque-scp/notifier#79]
The conditions for what constitutes an automatic subscription are complicated enough, but if I'm going to add Wikidot authoring and Crom as additional factors, it'll be almost too complex to document. 

I can ease the user experience by letting users request a list of exactly what they're subscribed to. 

Requesting a list should be entirely automated. One possible way for the user to communicate their intent would be to add a checkbox to their config. When notifier processes the request, it could edit the page and uncheck it. 

The list should contain information about the post or thread AND the reason that it is autosubscribed. The list might also contain autosubscribed posts that have been unsubscribed from too. 

I don't think there's much point batching requests until e.g. the daily channel. They should be processed in the next run. This would also make the feature appear responsive and would let users get and action the info faster. 

There might be some benefit to staggering how many requests can be processed in a single run, as a way to protect against an attack where many requests are made simultaneously. In that case, requests should be processed in page edit order, stalest first.

**Comment by rossjrw** [comment]
After #85 this will no longer be possible, as I will not be storing automatic subscriptions except for those that are actively emitting a notification. It will be pointless for a user to specifically request this list because it will be identical to the notification digest.

Closing as wontfix.
